plugins/outputs/kinesis/kinesis.go

```diff
@@ -169,23 +166,30 @@ func (k *KinesisOutput) Write(metrics []telegraf.Metric) error {
 
 	r := []*kinesis.PutRecordsRequestEntry{}
 
-	for _, p := range metrics {
+	for _, metric := range metrics {
 		atomic.AddUint32(&sz, 1)
```

Comment:
I know it was this way before this PR but is there a need for this to be atomic? Same for line 188.